addons/storysource/src/loader/generate-helpers.js

```diff
@@ -0,0 +1,106 @@
+import prettier from 'prettier';
+import { handleADD, handleSTORYOF } from './parse-helpers';
+
+const estraverse = require('estraverse');
+const acorn = require('acorn');
+
+require('acorn-stage3/inject')(acorn);
+require('acorn-jsx/inject')(acorn);
+require('acorn-es7')(acorn);
+
+const acornConfig = {
+  ecmaVersion: '9',
+  sourceType: 'module',
+  ranges: true,
+  locations: true,
+  plugins: {
+    jsx: true,
+    stage3: true,
+    es7: true,
+  },
+};
+
+function generateSourceWithoutUglyComments(source, comments) {
+  let lastIndex = 0;
+  const parts = [source];
+
+  const isUgly = comment => comment.value.trim().indexOf('eslint-') === 0;
+
+  comments.filter(isUgly).forEach(comment => {
+    parts.pop();
+
+    const start = source.slice(lastIndex, comment.start);
+    const end = source.slice(comment.end);
+
+    parts.push(start, end);
+    lastIndex = comment.end;
+  });
+
+  return parts.join('');
+}
+
+function prettifyCode(source) {
+  return prettier.format(source, {
```

Comment:
Can we make it optional and configurable?

Reply:
Only the prettier or the whole comments removal? Also. why?

Reply:
The prettier. Users may have their own preferences about how their code should be formatted

Reply:
Actually, I thought we won't expose it to users 😂 But I can make it configurable.

Reply:
> I thought we won't expose it to users

Why? I would like to use it in my project (and we don't use semicolons =)

Reply:
👌